### ISSUE TITLE
Support screen readers better (e.g. via ARIA)

### ISSUE THREAD

**Issue by Callek** [mozilla/build-relengapi#342]
So today, one of our blind employees found and tried to use the slave loan api.

In doing so he submitted 5 requests, and through a chat with :jdm in IRC I was told it was likely because the page has no interaction with screenreaders on if anything was submitted...

I was then pointed at  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions as a doc on how to make that better.

I consider missing assistive technology a bug, and important, though I doubt its only an issue in slave loan.

**Comment by djmitche** [comment]
I think Angular makes this a bit easier, and bootstrap has lots of support too.
